### PR TITLE
Improve logging of deployments

### PR DESCRIPTION
**Pull request by hjensas**
Currently inspector, api and INA all log as
"Deployment ironic ... : created", change this to log the actual name of the deployment

### CODE REVIEW

File: controllers/ironic_controller.go [hunk 1]
```diff
@@ -544,7 +544,7 @@ func (r *IronicReconciler) reconcileNormal(ctx context.Context, instance *ironic
 			return ctrl.Result{}, err
 		}
 		if op != controllerutil.OperationResultNone {
-			r.Log.Info(fmt.Sprintf("Deployment %s successfully reconciled - operation: %s", instance.Name, string(op)))
+			r.Log.Info(fmt.Sprintf("Deployment %s successfully reconciled - operation: %s", ironicNeutronAgenet.Name, string(op)))
```
Comment (abays) [inline controllers/ironic_controller.go:547]: `ironicNeutronAgenet` looks like a typo.  I know the variable is named the same above, but perhaps we could fix the spelling of `Agent` here and up there (L535).

Comment (hjensas) [reply controllers/ironic_controller.go:547]: Thanks, well spotted. I converted this to draft, and will fix the variable name as well.

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**